ZwiftPacketMonitor.Replay/DependencyRegistration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace ZwiftPacketMonitor.Replay
+{
+    class DependencyRegistration
+    {
+        public static ServiceProvider Register()
+        {
+            var serviceCollection = new ServiceCollection();
+
+            // added to allow logging level configuration, change MinLevel to LogLevel.Debug to see internal traces.
+            serviceCollection.AddLogging(configure => configure.AddConsole())
+                .Configure<LoggerFilterOptions>(configure => configure.MinLevel = LogLevel.Information);
+
+            serviceCollection.AddZwiftPacketMonitoring();
+            serviceCollection.AddSingleton<Replay>();
```

Comment:
I'm getting build errors here. Shouldn't this be `Replayer` and not `Replay`, given that the former is a type name, and the latter is a namespace?